safe/definitions/provenance.py

```diff
@@ -13,221 +13,226 @@
 }
 provenance_layer_aggregate_hazard_impacted = {
-    'key': 'provenance_layer_aggregate_hazard_impacted',
+    'key': u'provenance_layer_aggregate_hazard_impacted',
     'name': layer_purpose_aggregate_hazard_impacted['name'],
     'provenance_key': layer_purpose_aggregate_hazard_impacted['key']
 }
 provenance_layer_aggregation_summary = {
-    'key': 'provenance_layer_aggregation_summary',
+    'key': u'provenance_layer_aggregation_summary',
     'name': layer_purpose_aggregation_summary['name'],
     'provenance_key': layer_purpose_aggregation_summary['key']
 }
 provenance_layer_analysis_impacted = {
-    'key': 'provenance_layer_analysis_impacted',
+    'key': u'provenance_layer_analysis_impacted',
     'name': layer_purpose_analysis_impacted['name'],
     'provenance_key': layer_purpose_analysis_impacted['key']
 }
 provenance_layer_exposure_summary_table = {
-    'key': 'provenance_layer_exposure_summary_table',
+    'key': u'provenance_layer_exposure_summary_table',
     'name': layer_purpose_exposure_summary_table['name'],
     'provenance_key': layer_purpose_exposure_summary_table['key']
 }
 provenance_layer_profiling = {
-    'key': 'provenance_layer_profiling',
+    'key': u'provenance_layer_profiling',
     'name': layer_purpose_profiling['name'],
     'provenance_key': layer_purpose_profiling['key']
 }
 
 # Layers ID
 provenance_layer_exposure_summary_id = {
-    'key': 'provenance_layer_exposure_summary_id',
+    'key': u'provenance_layer_exposure_summary_id',
     'name': layer_purpose_exposure_summary['name'] + ' ID',
     'provenance_key': layer_purpose_exposure_summary['key'] + '_id'
 }
 provenance_layer_aggregate_hazard_impacted_id = {
-    'key': 'provenance_layer_aggregate_hazard_impacted_id',
+    'key': u'provenance_layer_aggregate_hazard_impacted_id',
     'name': layer_purpose_aggregate_hazard_impacted['name'] + ' ID',
     'provenance_key': layer_purpose_aggregate_hazard_impacted['key'] + '_id'
 }
 provenance_layer_aggregation_summary_id = {
-    'key': 'provenance_layer_aggregation_summary_id',
+    'key': u'provenance_layer_aggregation_summary_id',
     'name': layer_purpose_aggregation_summary['name'] + ' ID',
     'provenance_key': layer_purpose_aggregation_summary['key'] + '_id'
 }
 provenance_layer_analysis_impacted_id = {
-    'key': 'provenance_layer_analysis_impacted_id',
+    'key': u'provenance_layer_analysis_impacted_id',
     'name': layer_purpose_analysis_impacted['name'] + ' ID',
     'provenance_key': layer_purpose_analysis_impacted['key'] + '_id'
 }
 provenance_layer_exposure_summary_table_id = {
-    'key': 'provenance_layer_exposure_summary_table_id',
+    'key': u'provenance_layer_exposure_summary_table_id',
     'name': layer_purpose_exposure_summary_table['name'] + ' ID',
     'provenance_key': layer_purpose_exposure_summary_table['key'] + '_id'
 }
 provenance_layer_profiling_id = {
-    'key': 'provenance_layer_profiling_id',
+    'key': u'provenance_layer_profiling_id',
```

Comment:
Just curious, why do we need the `u`? It's only plain text.

Reply:
Ah, it is because I compare dictionary. This one was using string, the other one is using unicode. I forget that you have created that nice bytefy function, so I used it. But I forget to revert this changes. I will change it back like before.